proxyclient/m1n1/proxy.py

```diff
@@ -425,6 +425,12 @@ class IODEV(IntEnum):
     FB = 1
     USB0 = 2
     USB1 = 3
+    USB2 = 4
+    USB3 = 5
+    USB4 = 6
+    USB5 = 7
+    USB6 = 8
+    USB7 = 9
```

Comment:
We probably want to list the VUART here too. Can we move that one numerically before USB0? That way its value won't change as we add USB ports.

Reply:
we don't really interact with VUART iodev from python but done